src/mercury_engine_data_structures/formats/brfld.py

```diff
@@ -76,34 +86,32 @@ def follow_link(self, link: str) -> Any | None:
 
```

Comment:
`def follow_link(self, link: ActorLink) -> Any | None:`

Reply:
My reasoning was that an ActorLink refers specifically to an actor. The other methods will always return an actor specifically, but a generalized term is probably better, because it'll also accommodate future methods. I think I'm gonna go with `BrfldLink`.